src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,10 @@
+import { UserInfo } from '../UserInfo';
+
 // Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+export const TodoInfo = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed && ('TodoInfo--completed')}`}>
```

Comment:
this trick will not work with classes ${todo.completed && 'TodoInfo--completed'} - check your demo page and you will see false class many times
image
![image](https://github.com/mate-academy/react_static-list-of-todos-js/assets/6568745/909083a5-1e91-4919-b82f-32e78d913131)


So use classname library or use another approach